<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VSCRecordPlan</class>
 <widget class="QWidget" name="VSCRecordPlan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(41, 42, 43);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Editing</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(243, 243, 243);</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="radioButton">
          <property name="text">
           <string>All Day</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_2">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Week Day</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="VSCLineEdit" name="lineEdit">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>IBeamCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Template 01</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="VSCLineEdit" name="lineEdit_2">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(247, 247, 247);</string>
          </property>
          <property name="text">
           <string>Template 02</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="VSCLineEdit" name="lineEdit_3">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(247, 247, 247);</string>
          </property>
          <property name="text">
           <string>Template 03</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="VSCLineEdit" name="lineEdit_4">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(247, 247, 247);</string>
          </property>
          <property name="text">
           <string>Template 04</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="VSCLineEdit" name="lineEdit_5">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(247, 247, 247);</string>
          </property>
          <property name="text">
           <string>Template 05</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="VSCLineEdit" name="lineEdit_6">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(247, 247, 247);</string>
          </property>
          <property name="text">
           <string>Template 06</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="EditPlace" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="6" column="0">
         <widget class="QLabel" name="label_17">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Sat.</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_21">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Sun.</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_15">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Wed.</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_18">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Mon.</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_16">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Thu.</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_19">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Tue.</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_20">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Fri.</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="VSCPlanEditLine" name="Monwidget" native="true">
          <property name="minimumSize">
           <size>
            <width>648</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>648</width>
            <height>30</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="VSCPlanEditLine" name="Tuewidget" native="true">
          <property name="minimumSize">
           <size>
            <width>648</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>648</width>
            <height>30</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="VSCPlanEditLine" name="Wedwidget" native="true">
          <property name="minimumSize">
           <size>
            <width>648</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>648</width>
            <height>30</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="VSCPlanEditLine" name="Thuwidget" native="true">
          <property name="minimumSize">
           <size>
            <width>648</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>648</width>
            <height>30</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="VSCPlanEditLine" name="Friwidget" native="true">
          <property name="minimumSize">
           <size>
            <width>648</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>648</width>
            <height>30</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="VSCPlanEditLine" name="Satwidget" native="true">
          <property name="minimumSize">
           <size>
            <width>648</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>648</width>
            <height>30</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="VSCPlanEditLine" name="Sunwidget" native="true">
          <property name="minimumSize">
           <size>
            <width>648</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>648</width>
            <height>30</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="SaveButton">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="whatsThis">
        <string>Save </string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { 
border:none;
color:white;
background-image: url(:/action/resources/button.png);}
QPushButton:hover {
	background-image: url(:/action/resources/buttonpress.png);}
QPushButton:pressed {
	background-image: url(:/action/resources/button.png);}
</string>
       </property>
       <property name="text">
        <string> Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { 
border:none;
color:white;
background-image: url(:/action/resources/button.png);}
QPushButton:hover {
	background-image: url(:/action/resources/buttonpress.png);}
QPushButton:pressed {
	background-image: url(:/action/resources/button.png);}</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(41, 42, 43);</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Apply to</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Templates</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="ModComboBox">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Apply to Camera</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="IPCComboBox">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="OkPb">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { 
border:none;
color:white;
background-image: url(:/action/resources/button.png);}
QPushButton:hover {
	background-image: url(:/action/resources/buttonpress.png);}
QPushButton:pressed {
	background-image: url(:/action/resources/button.png);}
</string>
        </property>
        <property name="text">
         <string>Apply</string>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VSCPlanEditLine</class>
   <extends>QWidget</extends>
   <header>vscplaneditline.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VSCLineEdit</class>
   <extends>QLineEdit</extends>
   <header>vsclineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../main/vscmainwindows.qrc"/>
 </resources>
 <connections/>
</ui>
